orchestra/communication/staffing.py

```diff
@@ -148,3 +151,40 @@ def send_request_inquiries(staffbot, request, worker_batch_size):
              .format(request.task)))
         request.status = StaffBotRequest.Status.COMPLETE.value
         request.save()
+
+
+def get_available_requests(worker):
+    # We want to show a worker only requests for which there is no
+    # winner or for which they have not already replied.
+    won_responses = StaffingResponse.objects.filter(is_winner=True)
+    worker_provided_responses = StaffingResponse.objects.filter(
+        request_inquiry__communication_preference__worker=worker)
+    remaining_requests = (
+        StaffBotRequest.objects
+        .filter(inquiries__communication_preference__worker=worker)
+        .exclude(inquiries__responses__in=won_responses)
+        .exclude(inquiries__responses__in=worker_provided_responses)
+        .distinct())
+    inquiries = (
+        StaffingRequestInquiry.objects
+        .filter(request__in=remaining_requests)
+        .filter(communication_preference__worker=worker)
+        .order_by('request__task__start_datetime'))
+    # Because we might send multiple request inquiries to the same
+    # worker for the same request (e.g., email and slack), we
+    # deduplicate the inquiries so that we will return at most one
+    # inquiry's worth of content here.
+    request_ids = set()
```

Comment:
so we can't filter by a single communication method since they may have disabled it? unless we always force email i forget how we implemented it